packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -239,10 +252,33 @@ export interface IContainerRuntimeOptions {
     disableIsolatedChannels?: boolean;
 }
 
+/**
+ * Options for container runtime.
+ */
+export interface IContainerRuntimeOptions {
+    summaryOptions?: ISummaryRuntimeOptions;
+    gcOptions?: IGCRuntimeOptions;
+}
+
 interface IRuntimeMessageMetadata {
     batch?: boolean;
 }
 
+function localStorageEnableGC(): boolean | undefined {
```

Comment:
nit: The local storage flag can both enable and disable GC. Consider a neutral name for this - `localStorageRunGC`?